source/js/main.js

```diff
@@ -237,8 +237,10 @@ let main = {
     }
 
     // Embed additional instances of the Join Us box that don't need an API exposed (eg: Homepage)
-    if (document.querySelectorAll(`.join-us`)) {
-      var elements = Array.from(document.querySelectorAll(`.join-us`));
+    if (document.querySelectorAll(`.join-us:not(#join-us)`)) {
+      var elements = Array.from(
```

Comment:
We should be able to do this without Array "casting".

Reply:
what do you mean?

Reply:
fixed!